crates/engraver/src/render/input/chord.rs

```diff
@@ -250,7 +257,13 @@ impl Render for Chord {
             }
         };
 
-        Ok(Output { elements, width })
+        Ok(Output {
+            elements: vec![Element::Group {
+                id: self.id.clone(),
+                elements,
+            }],
+            width,
+        })
```

Comment:
Would it make any difference to your use case to *not* include the `<g>` wrapper if `id` isn't present? I kind of like the consistency of what you have here, but I guess it does add some size overhead to the SVG if all notes/rests/chords are wrapped in a potentially unnecessary tag. I could go either way on that.

Reply:
Yeah, absolutely. I actually thought about this while I was out on a walk this afternoon and was regretting I hadn't done it.